File: preview/understand/intent_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$intent_actions = $twilio->preview->understand->assistants("UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                              ->intents("tell-a-joke")
                                              ->intentActions()
                                              ->update(array(
                                                           "actions" => array(
                                                               "say" => "I was going to look for my missing watch, but I could never find the time."
                                                           )
                                                       )
                                              );

print($intent_actions->assistantSid);

File: autopilot/v1/task_actions/update-quickstart-example-1/update-quickstart-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Autopilot.V1.Assistant.Task;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var actions = new Dictionary<string, Object>()
        {
            {"say", "I was going to look for my missing watch, but I could never find the time."}
        };

        var taskActions = TaskActionsResource.Update(
            actions: actions,
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathTaskSid: "tell-a-joke"
        );

        Console.WriteLine(taskActions.AssistantSid);
    }
}
